lib/comgr/CMakeLists.txt

```diff
@@ -203,8 +218,8 @@ target_link_libraries(amd_comgr
     ${AMD_COMGR_PRIVATE_LINKER_OPTIONS}
     yaml-cpp
     ${CLANG_LIBS}
-    ${LLD_LIBS}
     ${LLVM_LIBS}
+    ${LLD_LIBS}
```

Comment:
LLD links against LLVM, and not the inverse, as far as I'm aware. Why change the order here?

Reply:
You're correct; this change is unnecessary. I've updated the PR to remove it.